# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/nsTransferableIPC.h
# Commit: 0df2f2832755
# Full Hash: 0df2f283275530fbfe5cf9c95986b073c0d6a31e
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2022-08-23 02:46:57
# Regressor Bug: 1781129
# File Overlap Count: 1
# Description:
#   Bug 1781129 - Part 1: Use BigBuffer for IPCDataTransfer, r=edgar
#   
#   The IPCDataTransfer type is used to transfer Clipboard/Drag & Drop payloads
#   over IPC to allow them to be written to or read from the relevant system
#   interfaces. Previously, the system which was used was somewhat complex, and
# ==============================================================================

diff -r 1d21d911b3e7 -r 0df2f2832755 widget/nsTransferableIPC.h
--- a/widget/nsTransferableIPC.h	Mon Aug 22 15:35:28 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,29 +0,0 @@
-/* -*- Mode: C++; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef nsTransferableIPC_h__
-#define nsTransferableIPC_h__
-
-#include "ipc/EnumSerializer.h"
-
-enum class TransferableDataType : uint8_t {
-  String,
-  CString,
-  InputStream,
-  ImageContainer,
-  Blob,
-};
-
-namespace IPC {
-
-template <>
-struct ParamTraits<TransferableDataType>
-    : public ContiguousEnumSerializerInclusive<TransferableDataType,
-                                               TransferableDataType::String,
-                                               TransferableDataType::Blob> {};
-
-}  // namespace IPC
-
-#endif  // nsTransferableIPC_h__
